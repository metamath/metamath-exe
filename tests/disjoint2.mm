$c formula |- ( ) $.

$v x y z w $.
xf $f formula x $.
yf $f formula y $.
zf $f formula z $.
wf $f formula w $.
combo $a formula ( x y ) $.
${
  $d x y $.
  ax-1 $a |- ( x y ) $.
$}

verybad $p |- ( ( x y ) ( z x ) ) $=
  xf yf combo zf xf combo ax-1 $.

${
  $d x y z $.
  stillbad $p |- ( ( x y ) ( z x ) ) $=
    xf yf combo zf xf combo ax-1 $.
$}

semigood $p |- ( ( x y ) ( z w ) ) $=
  xf yf combo zf wf combo ax-1 $.

${
  $d x y $.
  $d z w $.
  almostgood $p |- ( ( x y ) ( z w ) ) $=
    xf yf combo zf wf combo ax-1 $.
$}

${
  $d x y z w $.
  good $p |- ( ( x y ) ( z w ) ) $=
    xf yf combo zf wf combo ax-1 $.
$}

${
  $d x z $. $d x w $. $d y z $. $d y w $.
  stillgood $p |- ( ( x y ) ( z w ) ) $=
    xf yf combo zf wf combo ax-1 $.
$}
