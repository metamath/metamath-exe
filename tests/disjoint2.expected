MM> READ "disjoint2.mm"
Reading source file "disjoint2.mm"... 754 bytes
754 bytes were read into the source buffer.
The source has 33 statements; 2 are $a and 6 are $p.
No errors were found.  However, proofs were not checked.  Type VERIFY PROOF *
if you want to check them.
MM> Continuous scrolling is now in effect.
MM> 0 10%  20%  30%  40%  50%  60%  70%  80%  90% 100%
..................
?Error on line 15 of file "disjoint2.mm" at statement 12, label "verybad", type
"$p":
  xf yf combo zf xf combo ax-1 $.
                          ^^^^
There is a disjoint variable ($d) violation at proof step 7.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"( x y )" and "y" was substituted with "( z x )".
Variables "x" and "z" do not have a disjoint variable requirement in the
assertion being proved, "verybad".

?Error on line 15 of file "disjoint2.mm" at statement 12, label "verybad", type
"$p":
  xf yf combo zf xf combo ax-1 $.
                          ^^^^
There is a disjoint variable ($d) violation at proof step 7.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"( x y )" and "y" was substituted with "( z x )".  These substitutions have
variable "x" in common.

?Error on line 15 of file "disjoint2.mm" at statement 12, label "verybad", type
"$p":
  xf yf combo zf xf combo ax-1 $.
                          ^^^^
There is a disjoint variable ($d) violation at proof step 7.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"( x y )" and "y" was substituted with "( z x )".
Variables "y" and "z" do not have a disjoint variable requirement in the
assertion being proved, "verybad".

?Error on line 15 of file "disjoint2.mm" at statement 12, label "verybad", type
"$p":
  xf yf combo zf xf combo ax-1 $.
                          ^^^^
There is a disjoint variable ($d) violation at proof step 7.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"( x y )" and "y" was substituted with "( z x )".
Variables "x" and "y" do not have a disjoint variable requirement in the
assertion being proved, "verybad".
....
?Error on line 20 of file "disjoint2.mm" at statement 15, label "stillbad",
type "$p":
    xf yf combo zf xf combo ax-1 $.
                            ^^^^
There is a disjoint variable ($d) violation at proof step 7.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"( x y )" and "y" was substituted with "( z x )".  These substitutions have
variable "x" in common.
...
?Error on line 24 of file "disjoint2.mm" at statement 17, label "semigood",
type "$p":
  xf yf combo zf wf combo ax-1 $.
                          ^^^^
There is a disjoint variable ($d) violation at proof step 7.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"( x y )" and "y" was substituted with "( z w )".
Variables "x" and "z" do not have a disjoint variable requirement in the
assertion being proved, "semigood".

?Error on line 24 of file "disjoint2.mm" at statement 17, label "semigood",
type "$p":
  xf yf combo zf wf combo ax-1 $.
                          ^^^^
There is a disjoint variable ($d) violation at proof step 7.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"( x y )" and "y" was substituted with "( z w )".
Variables "w" and "x" do not have a disjoint variable requirement in the
assertion being proved, "semigood".

?Error on line 24 of file "disjoint2.mm" at statement 17, label "semigood",
type "$p":
  xf yf combo zf wf combo ax-1 $.
                          ^^^^
There is a disjoint variable ($d) violation at proof step 7.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"( x y )" and "y" was substituted with "( z w )".
Variables "y" and "z" do not have a disjoint variable requirement in the
assertion being proved, "semigood".

?Error on line 24 of file "disjoint2.mm" at statement 17, label "semigood",
type "$p":
  xf yf combo zf wf combo ax-1 $.
                          ^^^^
There is a disjoint variable ($d) violation at proof step 7.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"( x y )" and "y" was substituted with "( z w )".
Variables "w" and "y" do not have a disjoint variable requirement in the
assertion being proved, "semigood".
......
?Error on line 30 of file "disjoint2.mm" at statement 21, label "almostgood",
type "$p":
    xf yf combo zf wf combo ax-1 $.
                            ^^^^
There is a disjoint variable ($d) violation at proof step 7.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"( x y )" and "y" was substituted with "( z w )".
Variables "x" and "z" do not have a disjoint variable requirement in the
assertion being proved, "almostgood".

?Error on line 30 of file "disjoint2.mm" at statement 21, label "almostgood",
type "$p":
    xf yf combo zf wf combo ax-1 $.
                            ^^^^
There is a disjoint variable ($d) violation at proof step 7.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"( x y )" and "y" was substituted with "( z w )".
Variables "w" and "x" do not have a disjoint variable requirement in the
assertion being proved, "almostgood".

?Error on line 30 of file "disjoint2.mm" at statement 21, label "almostgood",
type "$p":
    xf yf combo zf wf combo ax-1 $.
                            ^^^^
There is a disjoint variable ($d) violation at proof step 7.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"( x y )" and "y" was substituted with "( z w )".
Variables "y" and "z" do not have a disjoint variable requirement in the
assertion being proved, "almostgood".

?Error on line 30 of file "disjoint2.mm" at statement 21, label "almostgood",
type "$p":
    xf yf combo zf wf combo ax-1 $.
                            ^^^^
There is a disjoint variable ($d) violation at proof step 7.  Assertion "ax-1"
requires that variables "x" and "y" be disjoint.  But "x" was substituted with
"( x y )" and "y" was substituted with "( z w )".
Variables "w" and "y" do not have a disjoint variable requirement in the
assertion being proved, "almostgood".
...................
All proofs in the database were verified.
